packages/office-ui-fabric-react/src/components/Panel/Panel.tsx

```diff
@@ -1,359 +1,11 @@
-                fontSize: IconFontSizes.large
-              },
-              rootHovered: {
-                color: theme.palette.neutralPrimary
-              }
-            }}
-            className={css('ms-Panel-closeButton ms-PanelAction-close')}
-            onClick={this._onPanelClick}
-            ariaLabel={closeButtonAriaLabel}
-            data-is-visible={true}
-            iconProps={{ iconName: 'Cancel' }}
-          />
-        </div>
-      );
-    }
-    return null;
-  };
-
-  private _onRenderHeader = (
-    props: IPanelProps,
-    defaultRender?: (props?: IPanelProps) => JSX.Element | null,
-    headerTextId?: string | undefined
-  ): JSX.Element | null => {
-    const { headerText, headerClassName = '' } = props;
-
-    if (headerText) {
-      return (
-        <div className={css('ms-Panel-header', styles.header)}>
-          <p
-            className={css('ms-Panel-headerText', styles.headerText, headerClassName)}
-            id={headerTextId}
-            role="heading"
-          >
-            {headerText}
-          </p>
-        </div>
-      );
-    }
-    return null;
-  };
-
-  private _onRenderBody = (props: IPanelProps): JSX.Element => {
-    const contentClass = css('ms-Panel-content', styles.content, props.isFooterAtBottom && styles.contentGrow);
-
-    return (
-      <div ref={this._content} className={contentClass} data-is-scrollable={true}>
-        {props.children}
-      </div>
-    );
-  };
-
-  private _onRenderFooter = (props: IPanelProps): JSX.Element | null => {
-    const { isFooterSticky } = this.state;
-    const { onRenderFooterContent = null } = this.props;
-    if (onRenderFooterContent) {
-      return (
-        <div className={css('ms-Panel-footer', styles.footer, isFooterSticky && styles.footerIsSticky)}>
-          <div className={css('ms-Panel-footerInner', styles.footerInner)}>{onRenderFooterContent()}</div>
-        </div>
-      );
-    }
-    return null;
-  };
-
-  private _updateFooterPosition(): void {
-    const _content = this._content.current;
-    if (_content) {
-      const height = _content.clientHeight;
-      const innerHeight = _content.scrollHeight;
-
-      this.setState({
-        isFooterSticky: height < innerHeight ? true : false
-      });
-    }
-  }
-
-  private _dismissOnOuterClick(ev: any): void {
-    const panel = this._panel.current;
-    if (this.state.isOpen && panel) {
-      if (!elementContains(panel, ev.target)) {
-        if (this.props.onOuterClick) {
-          this.props.onOuterClick();
-          ev.preventDefault();
-        } else {
-          this.dismiss();
-        }
-      }
-    }
-  }
-
-  private _onPanelClick = (): void => {
-    this.dismiss();
-  };
-
-  private _onTransitionComplete = (): void => {
-    this.setState({
-      isAnimating: false
-    });
-
-    if (!this.state.isOpen && this.props.onDismissed) {
-      this.props.onDismissed();
-    }
-  };
-}
+import { styled } from '../../Utilities';
```

Comment:
This file should be Panel.ts so Panel.base.tsx becomes a rename operation instead.